odl/tomo/backends/astra_setup.py

```diff
@@ -339,7 +347,7 @@ def astra_projection_geometry(geometry):
             'parallel3d', det_width, det_height, det_row_count,
             det_col_count, angles)
 
-    elif isinstance(geometry, CircularConeFlatGeometry):
+    elif False and isinstance(geometry, CircularConeFlatGeometry):
```

Comment:
Agree. `cone_vec` will be used for helical and circular acquisition curves. Vector arguments will be used accordingly for the parallel 3D and fanbeam case. However, the parallel 2d geometry does not support vector arguments according to the documentation.